src/components/MainPage.jsx

```diff
@@ -11,12 +11,22 @@ import BottomBar from "./BottomBar";
 import EventDetails from "./panels/EventDetails";
 import SpeakersPanel from "./panels/SpeakersPanel";
 import TalksPanel from "./panels/TalksPanel";
+import Navbar from "./Navbar";
+import mixpanel from 'mixpanel-browser'; // importing mixpanel
+
+
+mixpanel.init('YOUR_TOKEN'); // initializing mixpanel
```

Comment:
```suggestion
// mixpanel.init('YOUR_TOKEN'); // initializing mixpanel
```

Reply:
let's comment out this line for now since it's effectively a useless call until someone puts a token in